client/jetpack-cloud/sections/partner-portal/hooks.tsx

```diff
@@ -132,9 +131,7 @@ export function useLicenseIssuing(
 	const translate = useTranslate();
 	const dispatch = useDispatch();
 	const sites = useSelector( getSites ).length;
-	const products = useProductsQuery( {
-		select: selectAlphaticallySortedProductOptions,
```

Comment:
I moved this select function into `useProductsQuery` because every use of the hook used this select function, and the select function changes the return type which led to some confusion.